internal/services/gateway/action/org.go

```diff
@@ -117,6 +117,38 @@ func (h *ActionHandler) CreateOrg(ctx context.Context, req *CreateOrgRequest) (*
 	return org, nil
 }
 
+type UpdateOrgRequest struct {
+	Visibility cstypes.Visibility
```

Comment:
Should be a pointer and should update it only if nil. See other update apis like project.